docs/common/module/coffee-module-redispubsub.adoc

```diff
@@ -0,0 +1,368 @@
+        //logic
+        emitter.send(PubSubMessage.of("test", Map.of(LogConstants.LOG_SESSION_ID, "customSID")));
+    }
+}
+----
+
+== mp-reactive-messaging sajátosságok
+
+=== Azonos channel-re publisher és subscriber egy service-en belül
+
+Egy microservice-en belül a microprofile-reactive-messaging nem engedi, hogy publisher-t és subscriber-t is létrehozzunk
+ugyanarra a kulcsra, ha mégis ilyen igény merülne fel, akkor a `pub-sub-channel` attribútum használatával elválasztható
+a service-en belüli microprofile channel elnevezése és a hozzá kapcsolódó redis pub/sub channel neve, példa: <<common_module_coffee-module-redispubsub_same_pub_sub>>.
+
+=== Több producer használata azonos channel-en
+Alapértelmezés szerint egy channel-re a service-en belül egy helyről lehet üzenetet küldeni,
+ha több beanből szeretnénk, akkor a `mp.messaging.outgoing.test-out.merge=true` konfig aktiválásával tehetjük be.
+
+=== Konfigurációs kulcs megkötések
+
+Ha aktiválva van a microprofile-reactive-messaging subsytem és létezik mp-configban bármilyen `mp.messaging.*` akkor
+kell legyen a deploymentben hozzá tartozó subscriber vagy producer! Ez közös konfig fájlok esetén okozhat gondot.
+
+== Példák
+[#common_module_coffee-module-redispubsub_seperated_pub_sub]
+=== Subscriber és producer külön service-en
+
+image::seperated_pub_sub.drawio.svg[]
+
+==== Publisher
+.publisher config
+[source, yaml]
+----
+coffee:
+  redis:
+    sample: #<1>
+      database: 0
+      host: bs-sample-redis
+      port: 6379
+mp:
+  messaging:
+    outgoing:
+      test: #<2>
+        connector: coffee-redis-pubsub
+        connection-key: sample #<1>
+----
+<1> redis connection beállítás
+<2> emitter kulcsa
+
+.publishing method
+[source, java]
+----
+@Model
+public class TestAction {
+
+    @Inject
+    @Channel("test") //<1>
+    private Emitter<String> emitter;
+
+    void sendMessage() {
+        //logic
+        emitter.send("test123");
+    }
+}
+----
+<1> `mp.messaging.outgoing` kulcs
+
+==== Subscriber
+.config
+[source, yaml]
+----
+coffee:
+  redis:
+    sample: #<1>
+      database: 0
+      host: bs-sample-redis
+      port: 6379
+mp:
+  messaging:
+    incoming:
+      test: #<2>
+        connector: coffee-redis-pubsub
+        connection-key: sample #<1>
+----
+<1> redis connection beállítás
+<2> subscriber kulcsa
+
+.subscriber method
+[source, java]
+----
+@ApplicationScoped
+public class TestListener {
+
+    @Incoming("test") //<1>
+    void consume(String test) {
+        //logic
+    }
+}
+----
+<1> `mp.messaging.incoming` kulcs
+
+[#common_module_coffee-module-redispubsub_same_pub_sub]
+=== Subscriber és producer azonos service-en
+
+image::same_pub_sub.drawio.svg[]
+
+.config
+[source, yaml]
+----
+coffee:
+  redis:
+    sample: #<1>
+      database: 0
+      host: bs-sample-redis
+      port: 6379
+mp:
+  messaging:
+    incoming:
+      test-in: #<2>
+        connector: coffee-redis-pubsub
+        connection-key: sample #<1>
+        pub-sub-channel: test #<4>
```

Comment:
nem lattam leirasban ha nem adjuk ezt meg akkor mi a default ertek

Reply:
beírtam a fenti leírásba